docs/how_to_guides/msm_estimation_exercise.ipynb

```diff
@@ -0,0 +1,1419 @@
+{
```

Comment:
<p>Could you please provide a short explanation on how to read the plots. Just 2-4 sentences. No need to explain individual plots.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
Yes! I updated this section and added a few more explanations.